recipes/hy/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "hy" %}
+{% set version = "0.12.1" %}
+{% set md5 = "552e80e5ed1d32fd34f62ef53f78dbbc" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/46/00/b0a6855a11510b7d1f503ca723cfc2a411b940b9f9744c9b3f469533c92d/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  entry_points:
+    - hy = hy.cmdline:hy_main
+    - hy2 = hy.cmdline:hy_main
+    - hyc = hy.cmdline:hyc_main
+    - hyc2 = hy.cmdline:hyc_main
+    - hy2py = hy.cmdline:hy2py_main
+    - hy2py2 = hy.cmdline:hy2py_main
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - rply >=0.7.0
+    - astor >=0.5
+    - clint >=0.4
```

Comment:
Please drop any dependencies not needed at build time. Likely just `python` and `setuptools` are needed.